pkg/runtime/interfaces/propeller_executor_configuration.go

```diff
@@ -0,0 +1,11 @@
+// package interfaces
```

Comment:
@katrogan I'd like to abstract the config for propeller executor to a propeller executor config that contains both cluster configuration and namespace mapping configuration. This follows more of a OOP paradigm and I'm a little rusty with go, so could you recommend a way to make this as go idiomatic as possible.

Reply:
this looks fine!